lib/nimble_csv.ex

```diff
@@ -207,10 +228,10 @@ defmodule NimbleCSV do
     * `:dump_bom` - includes BOM (byte order marker) in the dumped document
     * `:reserved` - the list of characters to be escaped, defaults to the
       `:separator`, `:newlines`, and `:escape` characters above
-    * `:escape_formula` - the formula prefix(es) and formula escape sequence,
-       defaults to `nil`, which disabled formula escaping
-       `%{["@", "+", "-", "=", "\t", "\r"] => "'"}` would escape all fields starting
-       with `@`, `+`, `-`, `=`, tab or carriage return using the `'` character.
+    * `:escape_formula` - an optional map of formula prefixes to escape sequences.
+      When `nil` (the default), formula escaping is disabled. For example,
+      `%{~w(@ + - = \t \r) => "'"}` escapes fields starting with `@`, `+`, `-`, `=`,
+      tab, or carriage return by prefixing them with `'`
```

Comment:
Reworded this to improve flow. Similar to `:trim_bom`, `:escape_formula` requires more explanation so ends up being multiple sentences. Still, like all options, it doesn't end with punctuation.